applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -601,14 +602,19 @@ static void gnss_event_handler(int event)
 	}
 }
 
-/* Handles AT#XGPS commands. */
-int handle_at_gps(enum at_cmd_type cmd_type)
+AT_CMD_CUSTOM(xgps_set, "AT#XGPS=", handle_at_gps);
```

Comment:
Note: AT_CMD_CUSTOM does not correctly handle commands which start the same way, but differ in the end. For example AT#XGPS and AT#XGPSDEL both would call this method.

This will be fixed in the next libmodem release, after which this can only target AT#XGPS